slug_test.go

```diff
@@ -26,24 +26,29 @@ func TestPack(t *testing.T) {
 }
 
 func TestPack_rootIsSymlink(t *testing.T) {
-	err := os.Symlink("testdata/archive-dir", "testdata/archive-dir-symlink")
-	if err != nil {
-		t.Fatalf("Failed creating dir symlink: %v", err)
-	}
-	t.Cleanup(func() {
-		err := os.Remove("testdata/archive-dir-symlink")
+	for _, path := range []string{
+		"testdata/archive-dir",
+		"./testdata/archive-dir",
+	} {
+		symlinkPath := path + "-symlink"
+		err := os.Symlink(path, symlinkPath)
 		if err != nil {
-			t.Fatalf("failed removing testdata/archive-dir-symlink: %v", err)
+			t.Fatalf("Failed creating dir %s symlink: %v", path, err)
+
 		}
-	})
 
-	slug := bytes.NewBuffer(nil)
-	meta, err := Pack("testdata/archive-dir-symlink", slug, true)
-	if err != nil {
-		t.Fatalf("err: %v", err)
-	}
+		slug := bytes.NewBuffer(nil)
+		meta, err := Pack(symlinkPath, slug, true)
+		if err != nil {
+			t.Fatalf("err: %v", err)
+		}
 
-	assertArchiveFixture(t, slug, meta)
+		assertArchiveFixture(t, slug, meta)
+		err = os.Remove(symlinkPath)
```

Comment:
Curious: why remove the t.Cleanup wrapper around this?

Reply:
Otherwise it will complain that a symlink for that file already exists. There might be a better way to do this however